custom-recipes/pi-system-transpose/recipe.py

```diff
@@ -36,8 +37,8 @@ def parse_timestamp_and_value(line):
 
 def get_datetime_from_string(datetime):
     try:
-        time_stamp = dateutil.parser.isoparse(datetime)
-        return time_stamp
+        _ = dateutil.parser.isoparse(datetime)
+        return datetime
```

Comment:
No that's correct. It returns the date untouched if it conforms with the ISO format, None otherwise. The previous version was changing the format depending on pandas version.